gui/doc/viselements.json

```diff
@@ -479,6 +479,11 @@
             "name": "template[light]",
             "type": "dict",
             "doc": "The plotly layout template applied over the base template when theme is not dark."
+          },
+          {
+            "name": "decimator",
+            "type": "indexed(taipy.gui.data.Decimator)",
+            "doc": "A decimator instance for the indicated trace that will reduce the size of the data being sent back and forth.<br>If defined as indexed, it will apply the appropriate trace ; if not, it will apply only the the first trace."
```

Comment:
"it will apply the appropriate" -> "it will impact only the indicated"
" ; " -> "; ".